src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Dynamic Sampling
+sidebar_order: 6
+description: "Learn about the different methods for limiting the amount of data generated by your project."
+---
+
+
+TODO -  Should we use Dynamic Sampling or something like: "Centralized Sampling" or "Server Side Sampling" (I think
+that, from a user's perspective, "Dynamic Sampling" doesn't convey much information).
+
+
+
+# Why Dynamic Sampling
+
+Sentry SDKs are able to sample events by setting `traces_sample_rate` or `traces_sampler` during SDK initialisation
+(TODO add link to .../PerformanceMonitoring/SamplingTransactions).
+While this mechanism is very simple and works well in certain scenarios it has the downside of being difficult to
+change for scenarios where the application is distributed and redeployment is either not an option or very
+difficult (e.g. mobile applications).
+
+Dynamic Sampling addresses the need to easily change and manage what is sampled after an application is deployed,
+as an additional benefit Dynamic Sampling enables making more sophisticated sampling decisions that span multiple
+transactions.
+
+# Dynamic Sampling types
+
+Dynamic sampling operates on two types of events: errors and transactions.
```

Comment:
```suggestion
Dynamic sampling operates two both error events and transaction events.
```

Reply:
Shouldn't we have a preposition there ?
Maybe:
Dynamic sampling operates on both error events and transaction events.

Reply:
yes. Thanks, I dropped the preposition in my edit.